test/integration/functional_test.go

```diff
@@ -2064,6 +2066,30 @@ func startMinikubeWithProxy(ctx context.Context, t *testing.T, profile string, p
 	}
 }
 
+// validateExternalServiceAccess asserts that we can ping an external server from minikube.
+func validateExternalServiceAccess(ctx context.Context, t *testing.T, profile string) {
+	want := "Success Ping."
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(200)
+		_, err := w.Write([]byte(want))
+		if err != nil {
+			fmt.Fprintf(w, "Error: %v \n", err)
+		}
+	}))
+	defer srv.Close()
+
+	wgetStr := fmt.Sprintf("'wget --spider -S %v'", srv.URL)
+	rr, err := Run(t, exec.CommandContext(ctx, "kubectl", "--context", profile, "run", "--restart=Never",
```

Comment:
An easier way to get into a minikube cluster is `minikube ssh`

Reply:
I think the kubectl way is better, since he is doing it inside a pod, instead of inside minikube VM.